ui/nuxeo-aggregation/nuxeo-checkbox-aggregation.js

```diff
@@ -73,8 +195,85 @@ import { AggregationBehavior } from './nuxeo-aggregation-behavior.js';
     static get is() {
       return 'nuxeo-checkbox-aggregation';
     }
+
+    static get properties() {
+      return {
+        /**
+         * Make the element collapsible
+         */
+        collapsible: {
+          type: Boolean,
+          value: false,
+          reflectToAttribute: true,
+        },
+
+        /**
+         * Label of the element
+         */
+        label: {
+          type: String,
+          value: '',
+        },
+
+        /**
+         * Expanded or collapsed
+         */
+        opened: {
+          type: Boolean,
+          value: false,
+          reflectToAttribute: true,
+        },
+
+        _showAll: {
+          type: Boolean,
+          value: false,
+          readOnly: true,
+        },
+
+        _visibleBuckets: {
+          type: Array,
+          computed: '_computeVisibleBuckets(buckets, visibleItems, _showAll)',
+        },
+
+        /**
+         * If collapsible, the default visible items.
+         */
+        visibleItems: {
+          type: Number,
+          value: 8,
+        },
+      };
+    }
+
+    _computeVisibleBuckets(buckets, visibleItems, _showAll) {
+      let visibleBuckets = [];
+      if (!buckets || buckets.length === 0) {
+        return visibleBuckets;
+      }
+      visibleBuckets = _showAll ? buckets : buckets.slice(0, visibleItems);
```

Comment:
And we wouldn't need the visibleBuckets variable, and we could just return on this line.